tests/bgp/test_seq_idf_isolation.py

```diff
@@ -108,6 +108,7 @@ def test_idf_isolated_no_export(rand_one_downlink_duthost,
     nbrs = dut_nbrs(duthost, nbrhosts)
     orig_v4_routes = parse_routes_on_neighbors(duthost, nbrs, 4)
     orig_v6_routes = parse_routes_on_neighbors(duthost, nbrs, 6)
+    up_bgp_neighbors = duthost.get_bgp_neighbors_per_asic("established")
```

Comment:
Let's use config_reload(duthost, safe_reload=True, check_intf_up_ports=True, wait_for_bgp=True) to enable wait for bgp up after config reload

Reply:
Made the change as per your comments @abdosi @yejianquan. Validated that all the testcases are passing with the change